### ISSUE TITLE
Unable to use Predefined-Filters

### ISSUE THREAD

**Issue by zinka** [man-group/dtale#711]
Hi,

First of all, thank you very much for providing us with a great tool like Dtale. I have been using "predefined-filters" feature with older versions of Dtale. However, after upgrading to 2.9.0, I am unable to use this feature. When I click on predefined-filters in the left side menu-bar, I am going to blank page! Am I doing something wrong? Following is a simple code I am using:

import dtale.predefined_filters as predefined_filters
predefined_filters.set_filters([
    {
        "name": "Name",
        "column": "SBP",
        "description": "SBP Active",
        "handler": lambda df, val: df[(df["SBP"] == val) & (df["SBP"] > 2)],
        "input_type": "input",
        "default": 1,
        "active": True
    }
])

**Comment by aschonfeld** [comment]
@zinka I have figured out the issue with this filter (great catch BTW).  It was returning any value entered thru the frontend ton the backend as a string.  I have updated the frontend to correctly return numbers for integer or float columns.  But as a workaround until I can release a new version please update your code to the following:
```
import dtale.predefined_filters as predefined_filters
predefined_filters.set_filters([
{
"name": "Name",
"column": "SBP",
"description": "SBP Active",
"handler": lambda df, val: df[(df["SBP"] == int(val)) & (df["SBP"] > 2)],
"input_type": "input",
"default": 1,
"active": True
}
])
```

**Comment by aschonfeld** [comment]
@zinka just released v2.9.1 to pypi (will be on conda-forge shortly). Let me know if this update worked for you. Also, if you haven't already, please toss your ⭐ on the repo when you get a sec. Thanks 🙏
